csharp/src/Google.Protobuf.JsonDump/Google.Protobuf.JsonDump.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <EnvironmentFlavor>CLIENTPROFILE</EnvironmentFlavor>

```

Comment:
No, I don't think so. There may be all kinds of stuff in the build file that's left over from more complicated times. Will have a look at that at the same time.